lib/mayaUsd/ufe/UsdUndoAddNewPrimCommand.cpp

```diff
@@ -64,7 +64,7 @@ UsdUndoAddNewPrimCommand::UsdUndoAddNewPrimCommand(const UsdSceneItem::Ptr& usdS
 
         // The type of prim we were asked to create.
         // Note: "Def" means create typeless prim.
-        _primToken = (type.empty() || type == "Def") ? TfToken() : TfToken(name);
+        _primToken = (type.empty() || type == "Def") ? TfToken() : TfToken(type);

```

Comment:
No idea how this got left as "name" seeing as this problem was the first thing I discovered when implementing the new command.